starknet-rpc-test/src/utils.rs

```diff
@@ -130,6 +138,23 @@ impl AccountActions for SingleOwnerAccount<&JsonRpcClient<HttpTransport>, LocalW
         self.transfer_tokens_u256(recipient, U256 { high: FieldElement::ZERO, low: transfer_amount }, nonce)
     }
 
+    fn invoke_contract(
```

Comment:
let's call `invoke_contract` inside `transfer_tokens`

Reply:
Why?

transfer_token is a more specific version of invoke_contract

Reply:
Yes but at the end transfer_tokens is an invoke. All the code in `invoke_contract` is duplicated in `transfer_tokens`, why not just call it directly?